cpu/gd32v/periph/adc.c

```diff
@@ -122,7 +124,7 @@ int adc_init(adc_t line)
     }
     /* set clock prescaler to get the maximal possible ADC clock value */
     for (clk_div = 2; clk_div < 8; clk_div += 2) {
```

Comment:
I am wondering if I should have added an assert after this for-loop to make sure that the driver was able to honor the max freq.